src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -512,4 +513,23 @@ protected function getErrorMessageForException(Exception $e, array $messages)
             $e->getMessage()
         );
     }
+
+    protected function getBulkActionIds(Request $request, string $key): array
+    {
+        $ids = $request->request->get($key);
+
+        if (is_numeric($ids)) {
+            return [(int) $ids];
+        }
+
+        if (!is_array($ids)) {
+            return [];
+        }
+
+        foreach ($ids as $i => $id) {
+            $ids[$i] = (int) $id;
+        }
```

Comment:
```suggestion
        }
        
        // Return IDs ordered
        sort($ids);
```
Just a small improvement I recently added in product list, it doesn't change much but it avoids some confusing behaviours. For duplication, the products were not copied in the same order as the initially selected (because by default the list, which resulted in some change of order during bulk update.

I know this grid doesn't handle duplication, and it's a very small detail anyway but since this method will be shared by many controllers maybe this could be added here to keep consistency (especially since ProductController now relies on it).